src/components/index.ts

```diff
@@ -1,6 +1,5 @@
-import MucButton from './Button/MucButton.vue'
-import MucBanner from './Banner/MucBanner.vue'
-import MucIntro from './Intro/MucIntro.vue'
-import { MucCallout } from './Callout'
```

Comment:
You deleted your own new component here ;)

Reply:
Should we change the import "format" from the default-export of those specific vue-files/components to the module-import of the `index.ts` file inside those folders?

Reply:
Yes I think that would look cleaner!

Reply:
Issue above is resolved and import is changed to use moduled import for all known components until now.